instrumentation/netty-3.8/build.gradle

```diff
@@ -15,5 +15,5 @@ dependencies {
 }
 
 tasks.withType(Test) {
-    jvmArgs '-Dsplunk.context.server-timing.enabled=true'
+    jvmArgs '-Dsplk.context.server-timing.enabled=true'
```

Comment:
😭 Why??

Reply:
I wasn't around for this, but it _is_ the ticker symbol and it _is_ shorter.  😁 It's just unpronounceable.... 🤮